sqlx-core/src/postgres/listener.rs

```diff
@@ -65,8 +65,9 @@ impl PgListener {
 
     /// Starts listening for notifications on a channel.
     pub async fn listen(&mut self, channel: &str) -> Result<(), Error> {
+        // The channel name is quoted here to ensure case sensitivity
```

Comment:
I was thinking this should be part of the doc comment for `listen()` and listen_all()` since this is something that concerns the user.

Reply:
That's a good point.  Updated the pull request.  Let me know if that's what you had in mind.